src/content/docs/docs/evaluation.md

```diff
@@ -330,6 +330,27 @@ By default, `eval:run` runs against all configured evaluators, and as with
 `eval:flow`, results for `eval:run` appear in the evaluation page of Developer
 UI, located at `localhost:4000/evaluate`.
 
+### Batching evaluations
+
+:::note
+This feature is only available in the Node.js SDK.
+:::
+
+You can speed up evaluations by processing the inputs in batches using the CLI and Dev UI. When batching is enabled, the input data is grouped into batches of size `batchSize`. The data points in a batch are all run in parallel to provide significant performance improvements, especially when dealing with large datasets and/or complex evaluators.
+
+The `batchSize` option has been integrated into the `eval:flow` and `eval:run` CLI commands. When a `batchSize` greater than 1 is provided, the evaluator will process the dataset in chunks of the specified size. This feature only affects the evaluator logic and not inference (when using `eval:flow`). Here are some examples of enabling batching with the CLI:
```

Comment:
is 1 the default batch size? mention it?

Reply:
The default is no batching... I will mention it. Thanks!